### ISSUE TITLE
Scroll command is broken in webkit2

### ISSUE THREAD

**Issue by keis** [uzbl/uzbl#323]
the scroll command is broken in webkit2.

Is there a new api we can use to implement this?
Is it possible to add this functionality with the web-extension?

**Comment by mathstuf** [comment]
There may be some way to have JS do it, but then you get into the problem of web pages that hijack scrolling anyways. There may be some way to grab the scrolling bit of the webview widget through the GTK widget graph (there's a GTK widget inspector that may be of use).

**Comment by keis** [comment]
I think this might be what we want

https://webkitgtk.org/reference/webkitdomgtk/unstable/WebKitDOMDOMWindow.html#webkit-
dom-dom-window-scroll-by

**Comment by regnarg** [comment]
`scrollBy` seems to be a usable workaround:
~~~~~
@cbind <Up>             = js page string 'window.scrollBy(0, -100);'
@cbind <Down>           = js page string 'window.scrollBy(0, +100);'
@cbind <Left>           = js page string 'window.scrollBy(-100, 0);'
@cbind <Right>          = js page string 'window.scrollBy(+100, 0);'
@cbind k                = js page string 'window.scrollBy(0, -100);'
@cbind j                = js page string 'window.scrollBy(0, +100);'
@cbind h                = js page string 'window.scrollBy(-100, 0);'
@cbind l                = js page string 'window.scrollBy(+100, 0);'
@cbind <Page_Up>        = js page string 'window.scrollBy(0, -window.innerHeight);'
@cbind <Page_Down>      = js page string 'window.scrollBy(0, +window.innerHeight);'
~~~~~

**Comment by mathstuf** [comment]
That doesn't work if JavaScript is disabled.

**Comment by mathstuf** [comment]
Fixed by #365.
